operator/webhooks/pod_mutating_webhook.go

```diff
@@ -4,12 +4,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	appsv1 "k8s.io/api/apps/v1"
 	"net/http"
 	"reflect"
 	"strings"
 
 	"github.com/go-logr/logr"
-	klcv1alpha1 "github.com/keptn/lifecycle-toolkit/operator/api/v1alpha1"
+	kltv1alpha1 "github.com/keptn/lifecycle-toolkit/operator/api/v1alpha1"
```

Comment:
we had a discussion that this is the controllers api we are importing here so we can keep klcv1alpha1

Reply:
reverted